src/EnergyPlus/UnitarySystem.cc

```diff
@@ -4545,7 +4545,7 @@ namespace UnitarySystems {
                         this->m_CondenserNodeNum = thisCoolCoil.CondenserInletNodeNum(1);
 
                         if (this->m_FanExists) {
-                            thisCoolCoil.SupplyFanName = loc_m_FanName;
+                            thisCoolCoil.SupplyFanName = this->m_FanName;
```

Comment:
I ran the unit tests using a debug build and this line showed a memory access issue.